### PR TITLE
Front end/16 get information from sign inup page

### LINKED ISSUE #16: Get information from sign in/up page

### PR DESCRIPTION
**Pull request by julienlafargue**

### CODE REVIEW

File: flutter_application/lib/sign_up_page.dart [hunk 1]
```diff
@@ -22,31 +27,107 @@ class SignUpApp extends StatelessWidget {
...
-                  children: const <Widget> [
-                      SizedBox(height: 30,),
-                      Text("Create your count",
+                    crossAxisAlignment: CrossAxisAlignment.center,
+                    children: <Widget>[
+                      const SizedBox(
+                        height: 30,
+                      ),
+                      const Text(
+                        "Create your count",
```
Comment (Kubirt) [inline flutter_application/lib/sign_up_page.dart]: "Create your account"

File: flutter_application/lib/sign_up_page.dart [hunk 2]
```diff
@@ -22,31 +27,107 @@ class SignUpApp extends StatelessWidget {
...
+                      ),
+                      const SizedBox(height: 5,),
+                      Padding(
+                        padding: const EdgeInsets.symmetric(vertical: 8.0),
+                        child: Container( 
+                          width: 260,
+                          height: 60,
+                          child: TextField(
+                            onChanged: (value) {
+                                button.password = value;
```
Comment (Kubirt) [inline flutter_application/lib/sign_up_page.dart]: replace by button.email

File: flutter_application/lib/sign_up_page.dart [hunk 3]
```diff
@@ -22,31 +27,107 @@ class SignUpApp extends StatelessWidget {
...
+                      const SizedBox(height: 5,),
+                      Padding(
+                        padding: const EdgeInsets.symmetric(vertical: 8.0),
+                        child: Container( 
+                          width: 260,
+                          height: 60,
+                          child: TextField(
+                            obscureText: true,
+                            onChanged: (value) {
+                                button.email = value;
```
Comment (Kubirt) [inline flutter_application/lib/sign_up_page.dart]: replace by button.password

File: flutter_application/lib/sign_up_page.dart [hunk 4]
```diff
@@ -22,31 +27,107 @@ class SignUpApp extends StatelessWidget {
...
+                              border: OutlineInputBorder(
+                                borderRadius: BorderRadius.all(Radius.circular(8)),
+                              )
+                            ),
+                          ),
+                        ),
+                      ),
+                      const LoginAlready(),
+                      button,
+                      // ButtonConnection(
```
Comment (Kubirt) [inline flutter_application/lib/sign_up_page.dart]: remove useless comments

File: flutter_application/lib/tools/sign_up_tools.dart [hunk 5]
```diff
@@ -114,7 +35,7 @@ class ButtonConnection extends StatelessWidget {
         ),
         child: const Padding(
           padding: EdgeInsets.all(12.0),
-          child: Text('Connection',
+          child: Text('Create count',
```
Comment (Kubirt) [inline flutter_application/lib/tools/sign_up_tools.dart]: replace text by "create an account"

File: flutter_application/lib/tools/sign_up_tools.dart [hunk 6]
```diff
@@ -1,94 +1,15 @@
...
-      ),
-    );
-  }
-}
-
-class ButtonConnection extends StatelessWidget {
-  const ButtonConnection({super.key});
+class ButtonCreateAccount extends StatelessWidget {
+  ButtonCreateAccount({super.key, required this.firstname, required this.lastname,required this.email, required this.password});
+  String firstname;
```
Comment (Kubirt) [inline flutter_application/lib/tools/sign_up_tools.dart:9]: Unused variables ?

**Review by Kubirt — CHANGES_REQUESTED** [review]

**Review by Kubirt — APPROVED** [review]

**Review by Kubirt — APPROVED** [review]

**Review by UgoBoulestreau — APPROVED** [review]

**Pull request merged**